packages/node_modules/@ciscospark/internal-plugin-wdm/src/device/device.js

```diff
@@ -7,7 +7,9 @@ import {safeSetTimeout} from '@ciscospark/common-timers';
 import {omit} from 'lodash';
 import util from 'util';
 import FeaturesModel from './features-model';
+import ServiceCollection from './service-collection';
 import {persist, waitForValue, SparkPlugin} from '@ciscospark/spark-core';
+import * as Url from 'url';
```

Comment:
no need for `*` -> `import Url from 'url';`

Reply:
also, `url` is typically lowercase

Reply:
There are some use of `url` in the code so I had to change the name of the package to `Url` to prevent conflicts.